### DISCUSSION TITLE
NvChad Configuration & Setup

### DISCUSSION THREAD

**Discussion by tylsch** [scalameta/nvim-metals#604] [Q&A] [answered]
I'm just curious if anyone has gotten nvim-metals setup with NvChad.  I attempted to follow the setup here with no success https://github.com/folke/lazy.nvim/discussions/767

**Comment by tgodzik** [comment]
I transferred the discussion to nvim-metals, since I think it's more relevant here?

**Comment by ckipp01** [comment, marked as answer]
Hey @tylsch, we do have some users using NvChad. I know one just recently hit on some oddities and included their setup in https://github.com/scalameta/nvim-metals/issues/601. You should be able to use that as an example.

**Reply by tylsch** [reply]
Thanks.  I was able to get metals working from this thread.
